Improve wording in CoC regarding info to include in report. (#810)
writethedocs/www: docs/code-of-conduct.rst

In your report please include, when possible:

* Your contact info (so we can get in touch with you)

> * Names or descriptions of anyone who was involved or witness to the incident.

Reviewer: ```suggestion
* Names or descriptions of anyone who was involved or who witnessed the incident.
```

* When and where the incident occurred. Please be as specific as possible.
* Your account of what occurred. If there is a written record (e.g. tweets or slack messages) please include screenshots, or otherwise a link.
* Any extra context you believe existed for the incident.
* If you believe this incident is ongoing.
* Any other information you believe we should have.

> If you don’t have some of this information, or not at this time, please still make the report and include as much information as you have.

Reviewer: ```suggestion
If you don’t all of this information at the time, please still make the report and include as much information as you have.
```
